paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -191,22 +198,12 @@ internal fun AddCard(
                 .padding(bottom = 20.dp)
         )
 
-        PaymentMethodForm(
-            paymentMethodCode = PaymentMethod.Type.Card.code,
-            enabled = true,
-            onFormFieldValuesChanged = { },
-            completeFormValues = flowOf(),
-            hiddenIdentifiers = setOf(),
-            elements = listOf(
-                CardDetailsSectionElement(
-                    context = context,
-                    initialValues = mapOf(),
-                    viewOnlyFields = setOf(),
-                    identifier = IdentifierSpec.Generic("card_details"),
-                )
-            ),
-            lastTextFieldIdentifier = null,
-            modifier = Modifier.padding(bottom = 20.dp)
+        FormUI(
```

Comment:
Should we update `FormUI` (or similar) to accept `ViewData` instead? Maybe as a follow up?

Reply:
I looked into this, it is a bit more complicated than that. FormUI data is produced by both `FormController` and `FormViewModel`, so we would also need a `ViewData` flow for both producers.